Support/Scripts/prepare-android-toolchain.sh

```diff
@@ -11,7 +11,7 @@
 
 source "$(dirname "$0")/common.sh"
 
-[ $# -le 1 ] || die "usage: $0 [ARCH]"
+[ $# -le 2 ] || die "usage: $0 [ARCH] [PLATFORM]"
```

Comment:
We do because pre-21 we can only build arm.

On Mon, Aug 8, 2016 at 5:11 PM, Francis Ricci <notifications@github.com>
wrote:

> In Support/Scripts/prepare-android-toolchain.sh
> <https://github.com/facebook/ds2/pull/418#discussion_r73979325>:
>
> > @@ -11,7 +11,7 @@
> >
> >  source "$(dirname "$0")/common.sh"
> >
> > -[ $# -le 1 ] || die "usage: $0 [ARCH]"
> > +[ $# -le 2 ] || die "usage: $0 [ARCH] [PLATFORM]"
>
> So, we don't want to default to 21?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/ds2/pull/418/files/e248fd93289c97866b43137c53b6af07dfee8b19..891b41c62541fbda775166dfada5e546c9b3b8a0#r73979325>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AADHbNCXqdZ4GSNgfYxbtABhM_XTxla4ks5qd8W0gaJpZM4JffU1>
> .
>



-- 
Stephane Sezer